server/backend/api/app/constants.py

```diff
@@ -103,6 +104,19 @@ class EmailSendingStatus(str, Enum):
 APPLICATION_NAME_MAX_LEN = 100
 APPLICATION_DESC_MAX_LEN = 200
 CREATE_USER_MAX_LEN = 100
+MIN_PAGE = 1
+DEFAULT_PAGE_SIZE = 50
+MIN_PAGE_SIZE = 10
+
+# The intent is  to have "max=100" per page, however frontend is not ready, so if need to return "all records" found
+# and let frontend do the pagination, we could set it to 100000.
+# >> MAX_PAGE_SIZE = 100
+MAX_PAGE_SIZE = 100000
```

Comment:
This is temporary for 100000/per page since frontend isn't ready for backend pagination, so frontend can still get all records for now.